aldryn_people/boilerplates/bootstrap3/templates/aldryn_people/group_detail.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
     <div class="aldryn aldryn-people aldryn-group-detail">
         <h1>{% render_model group 'name' %}</h1>
-        {% for person in group.people.all %}
+        {% for person in group.people.all|dictsort:"name" %}
```

Comment:
Yes. In a future commit, name will be translated. And this will be rather complicated to sort by, without going to Python lists first.